Library/Formula/libxml++.rb

```diff
@@ -18,6 +16,21 @@ class Libxmlxx < Formula
 
   def install
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <libxml++/libxml++.h>
+
+      int main(int argc, char *argv[])
+      {
+         xmlpp::Document document;
+         document.set_internal_subset("homebrew", "", "http://www.brew.sh/xml/test.dtd");
+         xmlpp::Element *rootnode = document.create_root_node("homebrew");
+         return 0;
+      }
+    EOS
+    system ENV.cxx, "-I#{HOMEBREW_PREFIX}/include/libxml++-2.6", "-I#{HOMEBREW_PREFIX}/lib/libxml++-2.6/include", "-I#{HOMEBREW_PREFIX}/include/glibmm-2.4", "-I#{HOMEBREW_PREFIX}/lib/glibmm-2.4/include", "-I#{HOMEBREW_PREFIX}/include/glib-2.0", "-I#{HOMEBREW_PREFIX}/lib/glib-2.0/include", "-I#{HOMEBREW_PREFIX}/opt/gettext/include", "-I#{HOMEBREW_PREFIX}/include/sigc++-2.0", "-I#{HOMEBREW_PREFIX}/lib/sigc++-2.0/include", "-I/usr/include/libxml2", "test.cpp", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/opt/gettext/lib", "-L#{HOMEBREW_PREFIX}/lib", "-lxml++-2.6", "-lxml2", "-lglibmm-2.4", "-lgobject-2.0", "-lglib-2.0", "-lintl", "-lsigc-2.0", "-o", "test"
```

Comment:
You need to sort out the arguments. Please don't use `HOMEBREW_PREFIX`. Instead you should use `include`, `lib` and `Formula["foo"].opt_include`, `Formula["foo"].opt_lib`. And please remove all the unnecessary arguments such as `"-I#{HOMEBREW_PREFIX}/opt/gettext/include"`. Thanks.